apps/meteor/app/livechat/server/methods/sendMessageLivechat.ts

```diff
@@ -26,7 +26,7 @@ declare module '@rocket.chat/ui-contexts' {
 }
 
 export const sendMessageLivechat = async ({
-	message: { token, _id, rid, msg, file, attachments },
+	message: { token, _id, rid, msg, file, files, attachments },
```

Comment:
There is a place in the code, calling this function without all the properties, we need to take a look on that as well.

https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/livechat/server/methods/sendMessageLivechat.ts#L83

Reply:
I think nobody is using this Meteor method anymore... I've found zero locals searching... Removing it from code!

Reply:
We still need to add the `files` property to the method, we removed it and added it back, but we didn't add the missing `files` property there. 😬

Reply:
added